src/metricscollector/cmd/metricscollector/metricscollector_suite_test.go

```diff
@@ -64,6 +65,12 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 
 	ccNOAAUAA.RouteToHandler("GET", "/apps/an-app-id/containermetrics",
 		func(rw http.ResponseWriter, r *http.Request) {
+			if isTokenExpired {
+				isTokenExpired = false
```

Comment:
if token is expired,  NOAA will firstly refresh the token, and then call loggregator again with the refreshed token. If isTokenExpired is not reset to false, it will return 401 for second call to loggregator.